tests/testthat/test_helpers.R

```diff
@@ -35,3 +35,20 @@ test_that("list_submissions returns correct output", {
   out <- list_submissions(adc, '2018-10-01', '2018-10-03')
   expect_equal(out$submitter_name[1], 'Rachel Obbard')
 })
+
+test_that("recover a failed submission", {
```

Comment:
only one very minor suggestion - I think that the test description should be a little more grammatically correct